### DISCUSSION TITLE
Add Server Hubs

### DISCUSSION THREAD

**Discussion by payagej** [stoatchat/discussions#88] [Feature Suggestions]
I propose to be able to have "hub" servers for the different languages, so people could end up on servers with other people who speak their language.

**Comment by Darkflame72** [comment]
Could you not just have normal servers or server lists etc that provide this information? Having it as a feature for a specific use case would not be ideal I could imagine.

An alternative would be to have hubs that exist as public collections of servers but it would not be just for languages. This would allow someone to setup a Minecraft hub for example or a Python hub or even a Specific School hub.

**Reply by gdude2002** [reply]
I agree with this - it'd be fantastic to be able to have a public grouping of servers for specific topics, for example - although I'm really not sure on how it'd be implemented or shown UX-wise, I can see it being really handy.

**Comment by erkinalp** [comment]
I think they mean user-defined group directories.
